### DISCUSSION TITLE
git-resource to show commit message in resource version instead of commit hash

### DISCUSSION THREAD

**Discussion by vinay0013** [concourse/concourse#8908] [General]
Is there a concourse-resource which will show up the commit message in the resource version instead of the commit hash, so that it is more readable on the 1st look itself.

**Comment by amayer5125** [comment]
No, there is not a concourse-resource that will show the commit message in the resource version slot. The version needs to be unique, which is why they hash is used, so that Concourse knows when to trigger a new job. Commit messages are not guaranteed to be unique. As such, they would not make a good "version" to track.
